lib/svg.dart

```diff
@@ -692,6 +701,9 @@ class _SvgPictureState extends State<SvgPicture> {
   @override
   Widget build(BuildContext context) {
     Widget _maybeWrapWithSemantics(Widget child) {
+      if(_picture == null && widget.errorBuilder != null){
+        return widget.errorBuilder(context);
+      }
```

Comment:
This will never get called.  It's already wrapped in an `if (_picture != null)`.